plasmapy/tests/helpers/expected.py

```diff
@@ -0,0 +1,198 @@
+"""Contains a class to represent the expected outcome of a test."""
+
+__all__ = ["ExpectedTestOutcome"]
+
+import inspect
+
+from typing import Any
+
+from plasmapy.utils.code_repr import call_string
+
+
+def _is_warning(obj) -> bool:
+    """Return `True` if the argument is a warning, and `False` otherwise."""
+    return inspect.isclass(obj) and issubclass(obj, Warning)
```

Comment:
Curious - doesn't the latter imply the former? I mean, everything in Python derives from `Object`, no?